### ISSUE TITLE
Mining with Multi Wallets

### ISSUE THREAD

**Issue by Gortth** [trexminer/T-Rex#273]
Hello and thanks for the developpment and the follow up on TRex.
I've built a rig with and friend and I would like to know if it's possible to mine several wallets with dedicated pourcentage on each one ?
For example : DevTRex at 1%, me at 40% and my friend at 59%.
I've checked the TRex configuration and it seems that it's not possible for now.
Do you think this feature can be add for the next upcoming version of TRex ?
Thanks in advance

**Comment by trexminer** [comment]
You're right, it's not possible at the moment. There's very little demand for such a feature and there are a lot of bugs we need to fix before we have a chance to implement it. 
You can create a script than runs the miner for 40% of the day, terminates it, and then runs it with a different config for the rest of  the day. If you have more than a single card in your rig, you can also run two instances of T-Rex assigning GPUs according to your desired hashrate ratio.
